connectors/github/element-templates/github-connector.json

```diff
@@ -1153,6 +1182,67 @@
         ]
       }
     },
+    {
+      "label": "Git reference",
+      "description": "Branch or tag name for the workflow",
+      "group": "input",
+      "type": "String",
+      "feel": "optional",
+      "value": "",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "gitReference"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "createWorkflowDispatchEvent"
+        ]
+      }
+    },
+    {
+      "label": "Workflow ID",
+      "description": "ID or file name of the Github Actions workflow",
+      "group": "configuration",
+      "type": "String",
+      "feel": "optional",
+      "value": "",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "workflowId"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "createWorkflowDispatchEvent"
+        ]
+      }
+    },
+    {
+      "label": "Workflow inputs",
+      "description": "Key-value pairs in a single JSON object",
```

Comment:
[Note to reviewers](https://github.com/camunda/connectors/pull/1634#discussion_r1442975257) that was lost in rebase.